pyanaconda/payload/dnfpayload.py

```diff
@@ -380,12 +380,15 @@ def _add_repo(self, ksrepo):
             repo.exclude = ksrepo.excludepkgs
 
         if ksrepo.sslcacert:
+            log.repo(ksrepo.sslcacert)
```

Comment:
Are you sure this is correct?

Reply:
Good point, a debugging leftover. Removed. :)